service/src/main/java/uk/gov/hmcts/reform/fpl/service/orders/generator/ManageOrdersClosedCaseFieldGenerator.java

```diff
@@ -28,7 +28,7 @@
 
         Map<String, Object> data = new HashMap<>();
 
-        if (Order.isOrderFinal(order)) {
+        if (order.getIsOrderFinal()) {
```

Comment:
Just FYI: I think if you use boolean instead on Boolean, you might get a method called `isOrderFinal`.